services/leaf_space_integration/contact_reservations/models.go

```diff
@@ -4,7 +4,6 @@ type ContactReservation struct {
 	ID              string `json:"id"`
 	ContactStateId  string `json:"contactStateId"`
 	LeafspaceStatus string `json:"leafspaceStatus"`
-	Status          string `json:"status"`
```

Comment:
I checked again and you're right `Status` was replaced by `LeafspaceStatus`. And the name of model type is wrong because here the model is for defining the link between Leanspace contact status and LeafSpace contact status. Could you aslo replace the name `ContactReservation` by `ContactReservationStatusMapping`? Here but in other places too?

Reply:
I would rather change the name in the ticket with all modifications (since it's a breaking change): https://leanspace.atlassian.net/browse/GOLDS-379

Reply:
Ok sure. I will handle this ticket.